core/tee/tee_svc_cryp.c

```diff
@@ -499,6 +499,12 @@ struct attr_ops {
 	TEE_Result (*from_obj)(void *attr, void *src_attr);
 	void (*free)(void *attr);
 	void (*clear)(void *attr);
+	TEE_Result (*from_user_rsec)(void *attr, const void *buffer, size_t size);
```

Comment:
It's not clear to me when these functions will be used.

Reply:
Nevermind, found the explanation in https://github.com/OP-TEE/optee_os/issues/3287